account_payment_order/views/account_payment_order.xml

```diff
@@ -0,0 +1,279 @@
+                            string="Transfer Journal Entries"
+                            type="object"
+                            icon="fa-bars"
+                            attrs="{'invisible':[('move_ids', '=', [])]}"
+                        />
+                    </div>
+                    <div class="oe_title">
+                        <label for="name" class="oe_edit_only" />
+                        <h1>
+                            <field name="name" />
+                        </h1>
+                    </div>
+                    <group name="head" col="2">
+                        <group name="head-left">
+                            <field
+                                name="payment_mode_id"
+                                domain="[('payment_order_ok', '=', True), ('payment_type', '=', payment_type)]"
+                                widget="selection"
+                            />
+                            <field name="allowed_journal_ids" invisible="1" />
+                            <field
+                                name="journal_id"
+                                widget="selection"
+                                domain="[('id', 'in', allowed_journal_ids)]"
+                            />
+                            <field name="bank_account_link" invisible="1" />
+                            <field name="company_partner_bank_id" />
+                            <field
+                                name="company_id"
+                                groups="base.group_multi_company"
+                            />
+                            <field name="payment_type" invisible="0" />
+                            <field name="bank_line_count" />
+                        </group>
+                        <group name="head-right">
+                            <field name="date_prefered" />
+                            <field
+                                name="date_scheduled"
+                                attrs="{'invisible': [('date_prefered', '!=', 'fixed')], 'required': [('date_prefered', '=', 'fixed')]}"
+                            />
+                            <field name="date_generated" />
+                            <field name="generated_user_id" />
+                            <field name="date_uploaded" />
+                            <field name="description" />
+                        </group>
+                    </group>
+                    <notebook>
+                        <page name="payment-lines" string="Transactions">
+                            <field
+                                name="payment_line_ids"
+                                context="{'default_payment_type': payment_type}"
+                            />
+                        </page>
+                        <page
+                            name="bank-payment-lines"
+                            string="Bank Payment Lines"
+                            invisible="True"
```

Comment:
Thanks for adding the smart buttons! One of the benefits is that loading of the form of large orders (ours are > 20.000 lines) is faster when these large lists are not on loaded by default. You chose to keep them here, I see, marked as invisible which defeats that benefit. Would you consider removing them entirely?